src/fieldmap.c

```diff
@@ -48,6 +48,7 @@ static const struct MapConnection *GetIncomingConnection(u8 direction, int x, in
 static bool8 IsPosInIncomingConnectingMap(u8 direction, int x, int y, const struct MapConnection *connection);
 static bool8 IsCoordInIncomingConnectingMap(int coord, int srcMax, int destMax, int offset);
 
+
 #define GetBorderBlockAt(x, y)({                                                                   \
     u16 block;                                                                                     \
```

Comment:
?

Reply:
My mistake, I meant to copy a fix here, but went back on it. Will correct.